# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/uperf/hotfix.patch.cross
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- uperf-1.0.8/configure.ac.vanilla	2026-09-15 15:53:33.083330824 +0000
+++ uperf-1.0.8/configure.ac	2026-09-15 15:53:37.235317645 +0000
@@ -154,7 +154,7 @@
       AC_DEFINE([STRAND_THREAD_ONLY], [1], [Use only threads])
       AM_CONDITIONAL([HAVE_PROCESSES], [false])
 
-],[])
+],[process_shared_works=yes])
 # Checks for libraries.
 AC_MSG_CHECKING(whether to enable cpu performance counters support?)
 AC_ARG_ENABLE(cpc,
